source/geod24/LocalRest.d

```diff
@@ -1490,6 +1610,9 @@ unittest
 
         override void callFoo()
         {
+            if (!this.remote)
```

Comment:
Why has this changed? It doesn't work when it's in the constructor?

Reply:
it doesnt, `node_chn` gets initialized only after the first `spawn` finishes. Constructor runs before that.